LibraBFT/Impl/Consensus/BlockStorage/BlockTree.agda

```diff
@@ -30,10 +35,15 @@ postulate
 --   then Left  fakeErr
 --   else Right (lb & lbChildren %~ Set.insert hv)
 
+replaceTimeoutCertM : TimeoutCertificate → LBFT Unit
+replaceTimeoutCertM tc = do
+  lBlockStore ∙ bsInner ∙ btHighestTimeoutCert ?= tc
+  -- logInfo lTO (InfoUpdateHighestTimeoutCert (Just tc))
+
 ------------------------------------------------------------------------------
 
 insertBlockE : ExecutedBlock → BlockTree
-               → Either ErrLog (BlockTree × ExecutedBlock)
+             → Either ErrLog (BlockTree × ExecutedBlock)
```

Comment:
In the Haskell code, I have always tried to use `pure` (instead of `Just` or `Right`, etc).
But you are right: there are places in the Haskell codebase and the Agda codebase that are not consistent.
I just fixed two in Haskell and one in Agda.
I'll keep my eyes open for more - but not in this pull request.